tests/common-clustergroup-industrial-edge-hub.expected.yaml

```diff
@@ -368,6 +368,95 @@ data:
       git:
         repoURL: https://github.com/pattern-clone/mypattern
         revision: main
+    regionalDR:
+    - clusters:
+        primary:
+          install_config:
+            apiVersion: v1
+            baseDomain: example-domain.com
+            compute:
+            - name: worker
+              platform:
+                aws:
+                  type: m5.2xlarge
+                  zones:
+                  - eu-central-1a
+              replicas: 5
+            controlPlane:
+              name: master
+              platform:
+                aws:
+                  type: m5.2xlarge
+                  zones:
+                  - eu-central-1a
+              replicas: 3
+            metadata:
+              name: ocp-primary
+            networking:
+              clusterNetwork:
+              - cidr: 10.128.0.0/14
+                hostPrefix: 23
+              machineNetwork:
+              - cidr: 10.0.0.0/16
+              networkType: OpenShiftSDN
+              serviceNetwork:
+              - 172.30.0.0/16
+            platform:
+              aws:
+                region: eu-central-1
+                userTags:
+                  project: ValidatedPatterns
+            publish: External
+            pullSecret: ""
+            sshKey: ""
+          name: ocp-primary
+          version: 4.14.12
+        secondary:
+          install_config:
+            apiVersion: v1
+            baseDomain: fsi-partner.rhecoeng.com
```

Comment:
Replace baseDomain by "example-domain.com"

Reply:
Good catch!

Reply:
Fixed in https://github.com/RHEcosystemAppEng/regional-resiliency-pattern/pull/6/commits/cdf4acff07188bf8d49e80a4bab178ef3193ca6e.